ext/search_kit/ang/crmSearchTasks/crmSearchTasks.component.js

```diff
@@ -42,8 +42,16 @@
         return $scope.$eval('' + ctrl.ids.length + action.number);
       };
 
+      this.updateActionData = function() {
+        if (this.entity === 'RelationshipCache') {
+          this.entity = this.entityInfo.title = ts('Relationship');
```

Comment:
`this.entity` should *not* go through `ts()`

Reply:
Updated :+1: